change-beta/@azure-communication-react-83fe8c56-c493-4f0f-8165-c0c50b78b872.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
```

Comment:
I'm just wondering if it should be `none` instead

Reply:
This is what the `rush changelog` command gave me; if that's not what it should be then maybe this is a bigger discussion?

Reply:
It looks like it's internal changes that doesn't have any influence on customers, that's why I've proposed none instead of patch. I'm okay to leave it as `patch` if you feel that it's a better one.

Reply:
Then let's take this to the UI group and discuss.
The other PR is the same - `patch`
https://github.com/Azure/communication-ui-library/pull/3987/files#diff-08740a035ec35512b0f72a14ad52071108638c00aee665fa6ff19e3ba427b476